src/plugins/wpa/DataModel/QuicEvent.cs

```diff
@@ -125,36 +134,83 @@ public enum QuicEventId : ushort
         DatapathDestroyed,
     }
 
-    public abstract class QuicEvent : IKeyedDataType<Guid>
+    //
+    // The base class for all QUIC events.
+    //
+    public class QuicEvent : IKeyedDataType<Guid>
     {
-        public abstract Guid Provider { get; }
+        //
+        // Global configuration to control how parsing works. Defaults to WPA filter mode.
+        //
+        public static QuicEventParseMode ParseMode = QuicEventParseMode.WpaFilter;
```

Comment:
If you make this a property, you can remove 2211. You can preassign get and set properties just fine.

`public static QuicEventParseMode ParseMode { get; set; } = QuicEventParseMode.WpaFilter;`